posts/2022/09-24-coding-contentedly.md

```diff
@@ -0,0 +1,84 @@
+
+The lesson goes both ways as well. Overcommunicate emotion a little, and use emoticons when appropriate. Criticize code and compliment the author. Thank people and appreciate them publically when they make your life easier. Building up a bank of positivity will pay dividends next time you accidentally misstep at the keyboard.
+
+## Your code is not precious, and perfection is not the goal
+
+> "Murder your darlings"
+>
+> _-- Attributed to just about everyone everywhere_
+
+For the sake of your team, be willing to compromise on your code. Finding places to move forward and improve the codebase is more important than ideals. As soon as there is more than one person on a team, it is no longer possible for one person to be able to define what perfect is.
+
+Make things better one step at a time. The standard is not perfection but instead improving what we touch daily. Forget about the end goal and instead immerse yourself in the process. Enjoy the feeling of day-to-day craftsmanship and discard your dreams of the perfect codebase.  
+
+[Ego Is the Enemy is a great read](https://www.goodreads.com/book/show/27036528-ego-is-the-enemy)
+
+## Demonstrating beats dictating
+
+> “Be the change you want to see in the world.”
+>
+> _-- Arleen Lorrance_
+
+It's worth considering giving up on trying to convince people. Instead, show them. Don't use logic or arguments. Instead, demonstrate the benefits and see if they emulate it. I have, on multiple occasions, just started doing the right thing instead of wasting time and breath telling people they should be doing it. Nothing much happened for months, but the change was subtle and powerful. I gradually changed reality by doing it myself, and after many moons, it was no longer a new or outrageous idea. I had successfully normalized the idea, and others were willing to accept or try it themselves. Some people even told me they came up with the idea a few months later!
+
+A previous team lead phrased it as "be the change you wish to see in the world."
+
+And after all, if it is a good idea, it is still worth doing if only you are doing it.
+
+Attempting to change people is frustrating and pointless. Most of the time, you fail, and even if you manage to force your opinion on someone else, it can often leave them resentful and likely to drop it at the next opportunity.
+
+## Be kind and thank people
+
+It's easy to forget that people on their journeys surround us. Take the time to thank people when they do something good. It builds goodwill and encourages good behaviour. It costs nothing and just generally feels good. It practices gratitude which is enormous for preventing negative thinking. I think it might benefit the giver even more than the receiver.
```

Comment:
```suggestion
It's easy to forget that people on their journeys surround us. Take the time to thank people when they do something good. It builds goodwill and encourages good behaviour. It costs nothing and just generally feels good. It's a great way to practice gratitude, which is enormous for preventing negative thinking. I think it might benefit the giver even more than the receiver.
```